src/store/store.ts

```diff
@@ -0,0 +1,109 @@
+import { configureStore, combineReducers, Reducer } from "@reduxjs/toolkit";
+import {
+  FirebaseReducer,
+  firebaseReducer,
+  getFirebase,
+  actionTypes as reactReduxFirebaseActionTypes,
+} from "react-redux-firebase";
+import {
+  firestoreReducer,
+  constants as reduxFirestoreConstants,
+} from "redux-firestore";
+import LogRocket from "logrocket";
+
+import { Firestore } from "types/Firestore";
+import { User } from "types/User";
+
+import { worldUsersApi } from "./api/worldUsers";
+import { MiscReducers, VenueTemplateReducers } from "./reducers";
+
+export const rootReducer = combineReducers({
+  firebase: firebaseReducer as Reducer<FirebaseReducer.Reducer<User>>,
+  firestore: firestoreReducer as Reducer<Firestore>,
+  ...VenueTemplateReducers,
+  ...MiscReducers,
+  [worldUsersApi.reducerPath]: worldUsersApi.reducer,
```

Comment:
I chose to put it below so that it would 'win' if there were any clashes, but in reality, there should never be allowed to be any clashes here, and if there are, we will have much bigger problems. I don't feel like we need to do anything here.